cli/internal/util/env.go

```diff
@@ -0,0 +1,65 @@
+package util
+
+import (
+	"fmt"
+	"os"
+	"sort"
+	"strings"
+)
+
+// Prefixes for common framework variables that we always include
+var envVarPrefixes = []string{
+	"GATSBY_",
+	"NEXT_PUBLIC_",
+	"NUXT_ENV_",
+	"PUBLIC_",
+	"REACT_APP_",
+	"REDWOOD_ENV_",
+	"SANITY_STUDIO_",
+	"VITE_",
+	"VUE_APP_",
+}
+
+// getEnvPairsFromKeys returns a slice of key=value pairs for all env var keys specified in envKeys
+func getEnvPairsFromKeys(envKeys []string, allEnvVars []string) []string {
+	hashableConfigEnvPairs := []string{}
+	for _, envVar := range envKeys {
+		hashableConfigEnvPairs = append(hashableConfigEnvPairs, fmt.Sprintf("%v=%v", envVar, os.Getenv(envVar)))
```

Comment:
I was hoping to pull this out of `allEnvVars` instead of via `Getenv` here to reduce syscall count.